pliny-sidekiq.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "pliny", ">= 0.18.0"
-  spec.add_dependency "sidekiq", "<= 6.0.0"
+  spec.add_dependency "sidekiq", ">= 6.0.1", "< 7.0.0"
```

Comment:
Since we didn't touch `JobLogger#call`, technically this works with `<= 6.0.0` as well right?